### PR TITLE
Feauture/profile api implementation

### PR DESCRIPTION
**Pull request by dev-catalyst**
### Ticket
closes #

### Checklist
- [ ] There are no warnings in the browser console
- [ ] I made sure the component is WCAG compliant using ANDI
- [ ] I made sure the UI follows the style guide in the wireframe (font, font-size, etc...)
- [ ] I made sure that the screen reader can read all the text
- [ ] I made sure I removed all the console.logs
- [ ] I made sure mobile portrait view looks good
- [ ] I made sure mobile landscape view looks good
- [ ] I added storybook for the component
	- [ ] Storybook not applicable
- [ ] I added unit test coverage
	- [ ] I added integration test coverage
	- [ ] Unit test coverage not applicable

### To Do
- [ ]

### CODE REVIEW

**Review by cfernando415 — CHANGES_REQUESTED** [review]
pls see comment

**Comment by dev-catalyst** [comment]
All concerns are resolved.

File: src/components/Directory/BookDialog/index.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    setOpenConfirm(true);
+  };
+  const handleConfirmClose = () => {
+    setOpenConfirm(false);
+  };
+
+  // Session Dialog handlers
+  // const handleSessionOpen = () => {
+  //   setOpenSession(true);
+  // };
```
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Directory/BookDialog/index.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/151/commits/d792130f319cae1337b7751551c15b60ae41b756

File: src/components/Directory/BookDialog/index.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  // Meet time dialog handlers
+  // const handleMeetOpen = () => {
+  //   setOpenMeet(true);
+  // };
+  const handleMeetClose = () => {
+    setOpenMeet(false);
+  };
+
+  // Awesome dialog handlers
```
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Directory/BookDialog/index.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/151/commits/d792130f319cae1337b7751551c15b60ae41b756

File: src/components/Directory/BookDialog/index.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  const handleAwesomeOpen = () => {
+    setOpenAwesome(true);
+  };
+  const handleAwesomeClose = () => {
+    setOpenAwesome(false);
+  };
+
+  // Confirm Dialog handlers
```
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Directory/BookDialog/index.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/151/commits/d792130f319cae1337b7751551c15b60ae41b756

File: src/components/ViewContainer/index.js [hunk 4]
```diff
@@ -1,18 +1,73 @@
...
+    setOpenBackdrop(true);
+    try {
+      const res = await request.get(API_PATH.GET_MENTORS_PROFILEDATA);
+      if (res.data.data) {
+        var data = res.data.data;
+        setTimeSlots(data.timeSlot ? data.timeSlot : []);
+        setOpenBackdrop(false);
+      }
+    } catch (err) {
+      console.log(err);
```
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/ViewContainer/index.js:43]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/151/commits/fc70b4325b3d392792a9473fcbb706fc2c76b428

File: src/components/Settings/TimeSlotModal/index.js [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+      if (response.data.message === 'Success') {
+        setTimeSlots(response.data.data.timeSlot);
+        setOpenBackdrop(false);
+        setOpenModal(false);
+        setNotTimeSlot(false);
+      }
+    } catch (err) {
+      console.log(err);
+    }
+  };
```
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Settings/TimeSlotModal/index.js]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/151/commits/fc70b4325b3d392792a9473fcbb706fc2c76b428

File: src/components/Settings/AccountInfo.js [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+                inputProps: { 'data-testid': 'email' },
+                name: 'email',
+                autoFocus: true,
+                inputRef: register,
+                disabled: disabledInputs,
+                error: !isEmpty(errors.email),
+                helperText: errors.email && errors.email.message,
+              }}
+              // value={email}
+              // onChange={e => setEmail(e.target.value)}
```
[the remark this answers is not in the document]
Comment (cfernando415) [reply src/components/Settings/AccountInfo.js]: resolve in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/151/commits/d792130f319cae1337b7751551c15b60ae41b756

File: package.json [hunk 7]
```diff
@@ -12,8 +12,10 @@
     "focus-trap-react": "^8.5.0",
     "jwt-decode": "^3.1.2",
     "lodash": "^4.17.20",
+    "moment": "^2.29.1",
```
[the remark this answers is not in the document]
Comment (cfernando415) [reply package.json]: resolved in commit # https://github.com/femmecubator/femmecubator-ui-repo/pull/151/commits/0270f9d01db7048b1dea9c29f581144c7b52baeb

**Review by cfernando415 — APPROVED** [review]
:+1:

**Pull request merged**